MetadataExtractor/Formats/QuickTime/QuickTimeMetadataReader.cs

```diff
@@ -22,6 +23,9 @@ public static class QuickTimeMetadataReader
     {
         private static readonly DateTime _epoch = new DateTime(1904, 1, 1);
 
+        private static uint entryCount = 0;
+        private static readonly List<string> metaDataKeys = new List<string>();
```

Comment:
These cannot be static as it will not be safe for concurrent access.

Fixing this might require adding a data object to the `QuickTimeReader.ProcessAtoms` API so we can flow such data around. There may be an easier way of fixing this however.

Reply:
Yup, I called that out in the issue discussion thread before creating the PR.
Adding a data object `QuickTimeReader.ProcessAtoms` may be the easiest solution. 
Initially I did not want to mess with / clutter that, but I think it will work. I will update the PR. Thanks.